omod/src/main/resources/config.xml

```diff
@@ -52,5 +52,31 @@
 	</messages>
 	<!-- /Internationalization -->
 
+	<!-- Global Properties Required by the Order Entry OWA -->
+	<globalProperty>
+		<property>orderentryowa.encounterType</property>
+		<description>Order Entry Encounter Type</description>
+		<defaultValue>Order Entry</defaultValue>
```

Comment:
Is there a reason why you are using name instead of uuid?

Reply:
Thannks @dkayiwa ,
i ddnt intend to make any changes to the[ OWA](https://github.com/openmrs/openmrs-owa-orderentry#usage), 
so i simply folowed the instructions of what the setting `orderentryowa.encounterType` , should be .
From the instructions given , its supposed to be the name of the Encounter Type